notebook/static/notebook/js/actions.js

```diff
@@ -423,6 +415,21 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
+        'toggle-toolbar':{
+            help: 'hide/show the toolbar',
+            handler : function(env){
+                $('div#maintoolbar').toggle();
```

Comment:
Hmm, that is misleading.  I would have thought `env` === `Jupyter` seeing that we access `env.notebook`.